mkdocs.yml

```diff
@@ -3,6 +3,8 @@ site_url: https://github.com/PrairieLearn/PrairieLearn
 repo_url: https://github.com/PrairieLearn/PrairieLearn
 theme: readthedocs
 docs_dir: doc
+# Fix "Edit on GitHub" link: (defaults to edit/master/docs/, note the s)
+edit_uri: blob/master/doc/
```

Comment:
Should this be `edit/master/doc/`?

Reply:
`edit/master/doc/` will lead directly to the page source opened in the web editor, whereas `blob/master/doc/` leads to the viewing page instead (like normal browsing on GitHub would). My thought was that this would be more appropriate if someone just wanted to locate the source file on GH quickly, and also it doesn't require the user to be logged in to work.

Reply:
Of course someone can click the normal GitHub edit button after arriving at the viewing page, too.

Reply:
Sounds good to me, thanks!